### PR TITLE
added comment support, fixed enabled state update

### PR DESCRIPTION
**Pull request by evgsil**
It is crucial for me to have ability to specify distribution comment. It allows you easily mark and recognize distributions for different projects, stages etc.

Also fixed situation when distribution should be enabled/disabled without other changes

### CODE REVIEW

**Review by eahefnawy — APPROVED** [review]
Thanks @evgsil ... looks good, would love some extra eyes from @danielcondemarin here 😊

File: serverless.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ class CloudFront extends Component {
     this.context.status('Deploying')
 
     inputs.region = inputs.region || 'us-east-1'
+    inputs.enabled = inputs.enabled === false ? false : true
+    inputs.comment =
+      inputs.comment === null || inputs.comment === undefined ? '' : String(inputs.comment)
```
Comment (danielcondemarin) [inline serverless.js]:
```suggestion
      typeof inputs.comment === 'string' ? inputs.comment : ''
```

**Review by danielcondemarin — APPROVED** [review]
~~Just a tiny suggestion, but aside from that~~

LGTM 🚀

File: serverless.js (hunk 1 above)
Comment (evgsil) [reply serverless.js]: My intention was to leave possibility to put numbers or arrays in comments. It is really easy to forget that your need to quote numbers and in such situation your comment just will not appear

Comment (danielcondemarin) [reply serverless.js]: That’s fair enough 🙂

**Review by danielcondemarin — APPROVED** [review]

**Pull request merged**